proprietary/tokens/README.md

```diff
@@ -1,3 +1,229 @@
+
+Some examples:
+
+```css
+:root {
+  --ams-color-blue: #004699;
+  --ams-text-font-weight-bold: 800;
+  --ams-border-width-medium: 0.125rem;
+  --ams-button-padding-inline: 1.5rem;
+  --ams-space-xl: clamp(2.25rem, 2.0625rem + 0.9375vw, 3rem);
+  --ams-text-input-invalid-border-color: var(--ams-color-red);
+}
+```
+
+Tokens start their name with a prefix of `ams-` in CSS.
+The name of a component token (see below) ends with the property that uses it, e.g. `-font-size`.
+
+All tokens together form a theme that encodes the entire branding of the City of Amsterdam.
+
+## Installation
+
+Use only this package if you want or need to reference the tokens directly in your project.
+
+Note that our [CSS components](https://www.npmjs.com/package/@amsterdam/design-system-css) and [React components](https://www.npmjs.com/package/@amsterdam/design-system-react) provide more functionality and apply the tokens automatically.
+You should use these packages if your application uses React, or if it allows applying our HTML classes.
+
+```sh
+npm i @amsterdam/design-system-tokens
+```
+
+```sh
+pnpm add @amsterdam/design-system-tokens
+```
+
+```sh
+yarn add @amsterdam/design-system-tokens
+```
+
+## Usage in CSS
+
+Tokens are typically used as custom properties in CSS.
+
+### Main stylesheet
+
+This package offers a main stylesheet containing all tokens.
+They are declared as global CSS variables through the `:root` selector.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+```
+
+If that doesn’t work for your project, use `index.theme.css` instead, which uses an `.ams-theme` selector.
+Add that class to a root element of your application.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.theme.css";
+```
+
+```html
+<html class="ams-theme">
+  …
+</html>
+```
+
+### Compact mode
+
+Our branding is rather spacious for websites.
+The main stylesheet implements this ‘spacious mode’ as the default.
+A compact stylesheet is available for applications that require less white space and smaller text.
+
+Note that the compact stylesheet is not independent – it only contains overrides.
+Import it after the main stylesheet for the correct result.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+import "@amsterdam/design-system-tokens/dist/compact.css";
+```
+
+### Three layers
+
+The tokens are organised in three layers: brand, common and component tokens.
+
+#### Brand tokens
+
+These express the corporate identity of the City of Amsterdam.
+They are our fundamental selection from all possible colours, text characteristics, spacing lengths, border widths, etc.
+Examples:
+
+```css
+:root {
+  --ams-color-primary-red: #ec0000;
+  --ams-space-md: 1rem;
+  --ams-proportion-wide: 4/3;
+  --ams-border-width-lg: 0.1875rem;
+}
+```
+
+Brand tokens must only be referenced by common tokens; not by component tokens or component stylesheets.
+(We are currently working to apply this rule to our own libraries.)
```

Comment:
This is still confusing imo. We seem to be telling consumers of our tokens package that they shouldn't use brand tokens and later on that they shouldn't use component tokens. The common tokens layer doesn't have everything in brand and components yet (and I'm honestly not sure they ever will), so then consumers are out of luck? 

I think we want to encourage consumers to use brand tokens, right? The text you wrote here might become an internal guideline, but I don't think it's relevant for external users.

Reply:
This may need more thought. I’ve deleted it to allow this PR to go through.